frontend/src/views/ProducerFormPage/index.vue

```diff
@@ -50,6 +50,7 @@
         :tab-titles="titles"
         v-model="selectedTabIndex"
         @update:modelValue="selectTab"
+        class="allow-overflow"
```

Comment:
Il y a aussi la classe Tailwind `overflow-visible` : https://v3.tailwindcss.com/docs/overflow#showing-content-that-overflows

Reply:
merci - je viens de le tester et ça ne marche pas dans ce cas. Je sais pas pourquoi mais le CSS est filtré quand c'est la class tailwind, et pas quand c'est notre classe